lib/src/logging.rs

```diff
@@ -57,7 +57,10 @@ impl LogEndpoint {
         }
         to_write.push(b'\n');
 
-        LOG_WRITER.lock().unwrap().write_all(&to_write)
+        eprintln!("locking");
```

Comment:
Were these prints meant to be in the final product?

Reply:
Nope, thank you for catching those!